docs/tutorials/runtime-module-in-own-crate.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Write a Runtime Module in its Own Crate"
+---
+
+In this tutorial you'll write substrate runtime module that lives in its own crate, and include it in a node based on the node-template. If you prefer you can follow along with the [video walkthrough](todo record after initial feedback).
+
+## Setup a Development Environment
+If you haven't already, follow the guide to [install substrate](../getting-started/installing-substrate). You'll need the substrate-up scripts, but not the standard substrate node, so the `--fast` option is what you want here.
+
+## Create a new Node Template
+We'll begin by creating a brand new node template.
+```bash
+substrate-node-new modular-chain <your name>
+```
+If you've followed other runtime tutorials, you've probably started this way.
+
+In this tutorial, we're not going to write our module directly as part of the node template, but rather as a separate rust crate. This approach provides a few advantages:
+1. Our module can be easily imported into other nodes in the future.
+2. Our module can be published to places like github or crates.io without publishing our entire node in the same place.
+
+We do still need the node template, however, to actually run our module.
+
+## Create a Rust Library Crate
+So if our module isn't going to live in the node template, where _is_ it going to live? In a rust library crate. We can create a blank one with
+```bash
+cargo new --lib test_module
+```
+
+If you aren't familiar with library crates or how to use them in other rust projects, it's probably worth reading the [cargo docs](https://doc.rust-lang.org/cargo/guide/creating-a-new-project.html). In the video walkthrough of this tutorial, I briefly describe a simple number doubling program that uses a library crate for the doubling.
+
+## Make the Crate a Runtime Module
+The cargo command has created the appropriate files and directories for us.
+Let's replace the contents of `lib.rs` in our new crate with the module template from `runtime/template.rs`. (You can also use `substrate-module-new` to create such a file.)
+
+Go ahead and compile your new crate with `cargo build`. It won't compile yet, because that template module has some dependencies from substrate itself. The compiler errors give you clues about what you need to include. We can just copy the relevant dependencies from the original node template runtime's `Cargo.toml`. Add these lines to your crate's `Cargo.toml`.
```

Comment:
There is an existing `[dependencies]` tag which you should mention to remove?

Reply:
Irrelevant now that I'm using the template